pkg/apis/eventing/v1alpha1/trigger_defaults_test.go

```diff
@@ -27,15 +27,19 @@ var (
 	defaultBroker = "default"
 	otherBroker   = "other_broker"
 
+	binaryEncoding     = false
```

Comment:
at what version?

Reply:
just named constants in the test vs using true/false in places. Can rename if helpful.